lib/src/parse/stylesheet.dart

```diff
@@ -503,8 +503,11 @@ abstract class StylesheetParser extends Parser {
   /// This is only used in contexts where declarations are allowed but style
   /// rules are not, such as nested declarations. Otherwise,
   /// [_declarationOrStyleRule] is used instead.
-  @protected
```

Comment:
Does this no longer apply?

Reply:
I'm not sure why it was there in the first place—the method is private, so it can't be referenced from subclasses anyway (unless those subclasses are in the same file which they aren't).